### ISSUE TITLE
AVT1 Violation in GaugeChart component

### ISSUE THREAD

**Issue by RakshaFauzdar** [carbon-design-system/carbon-charts#1825]
### Application/Team

Vajra

### Environment

Browser

### What happened? What did you expect to happen instead?

<img width="1237" alt="Screenshot 2024-05-21 at 3 10 20 PM" src="https://github.com/carbon-design-system/carbon-charts/assets/71915465/16b3ae87-0109-41cb-a50b-07b782b4ba79">

I'm facing this accessibility issue when i'm using <GaugeChart/> which is imported from @carbon/charts-react
this avt1 issue comes up whenever we are using additional item for legend.
`legend: {
                enabled: true,
                additionalItems: [
                    {
                        
                        type: 'area',
                        name: 'Not scanned',
                        width: `${BLOCK_CLASS_NAME}--legendSize`,
                    },
                ],
}`

### What WCAG 2.1 checkpoint does the issue violate?

AVT1

### Version

"@carbon/charts-react": ^1.15.6

### What priority level would this issue be in your opinion?

P1

**Comment by RakshaFauzdar** [comment]
Can anyone please provide any update or fix solution  so that we can use to make legends in guage charts hoverable
